exec/java-exec/src/test/java/org/apache/drill/exec/expr/fn/FunctionInitializerTest.java

```diff
@@ -45,25 +45,29 @@
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
-import static org.mockito.Matchers.any;
-import static org.mockito.Mockito.doAnswer;
-import static org.mockito.Mockito.spy;
 
-@RunWith(MockitoJUnitRunner.class)
 @Category(SqlFunctionTest.class)
 public class FunctionInitializerTest {
 
-  private static final String CLASS_NAME = "com.drill.udf.CustomLowerFunction";
+  @ClassRule
+  public static final BaseDirTestWatcher dirTestWatcher = new BaseDirTestWatcher();
```

Comment:
Sorry, missed that in my previous review. Consider using `TemporaryFolder` rule instead of `BaseDirTestWatcher`.